query-algebrizer/Cargo.toml

```diff
@@ -6,6 +6,7 @@ workspace = ".."
 [dependencies]
 enum-set = { git = "https://github.com/rnewman/enum-set" }
 error-chain = "0.8.1"
+maplit = "0.1"
```

Comment:
Can this be a build-only dependency?  Having investigated the path to landing in Firefox more, I think we'll need to stop using some of our dependencies pretty soon to constrain our code size.